docs-src/references/strongly-typed-errors/cause-unhandled-update.md

```diff
@@ -0,0 +1,15 @@
+---
+id: cause-unhandled-update
+title: Cause Unhandled Update
+description: Explanation for Cause Unhandled Update error message.
+sidebar_label: Cause Unhandled Update
+tags:
+  - errors
+  - strongly-typed
+---
+
+`UnhandledUpdate` occurs when a Workflow Update is received by the server while a Workflow Task is being processed on a Worker producing a command that would cause the Workflow to transition to a closed state.
+
+Temporal rejects the Workflow Task completion in order to guarantee that Updates are eventually seen by Workflow code and will rewind the Workflow so it can handle the pending Update.
```

Comment:
```suggestion
Temporal rejects the Workflow Task completion to guarantee that the update is eventually handled by Workflow code and rewinds the Workflow so it can handle the pending update.
```